llvm/include/llvm/Analysis/VectorUtils.h

```diff
@@ -195,6 +195,11 @@ static constexpr char const *MappingsAttrName = "vector-function-abi-variant";
 /// the presence of the attribute (see InjectTLIMappings).
 void getVectorVariantNames(const CallInst &CI,
                            SmallVectorImpl<std::string> &VariantMappings);
+
+/// Constructs a FunctionType by applying vector function information to the
+/// type of a matching scalar function.
+FunctionType *createFunctionType(const VFInfo &Info,
```

Comment:
nit: since this is going to be publicly available interface, I think it is worth to use doxygen style to document the parmeters.